### ISSUE TITLE
YoloV2 training on Small Images 

### ISSUE THREAD

**Issue by sayanmutd** [pjreddie/darknet#469]
How would i train to detect small characters on size of images 128*72. When i am training on this dataset after sometime the value of IOU is becoming nan. I am using tiny-yolo as my model.  How would i do that?
@pjreddie @lilohuang

**Comment by Laranto** [comment]
Can you give us your .cfg file?

**Comment by sayanmutd** [comment]
yeah it is same as tiny-yolo.cfg with classes=1 and input size 128*72

**Comment by sayanmutd** [comment]
Do i need to change the grid size, as this small input images contains 10 non overlapping components which belongs to the same class.

**Comment by sayanmutd** [comment]
The training is continously identifying wrong classes and the count is sometimes showing above 30 or 40 wherein it originally has 10 labels

**Comment by sayanmutd** [comment]
I have resized the image but still there are no bounding boxes. I am only receiving the class labels.

**Comment by sayanmutd** [comment]
@AlexeyAB I have changed the input size to the multiple of 32 as per yolov2 requirements but still i am only getting the class labels and no detection and i have changed the number of anchors to 1 as there are non overlapping objects which fall in the same category to detect. Please help me out of this situation

**Comment by TheMikeyR** [comment]
I've read that the image input resolution should be the same size as the network (cfg) resolution.

**Comment by sayanmutd** [comment]
I have one way resolved this issue by giving the test image 800*600

**Comment by AlexeyAB** [comment]
@sayanmutd 

* What command do you use for detection?
* What anchor values do you use when you changed the number of anchors to 1?
* Can you show your cfg-file?

**Comment by Venus24kk** [comment]
Hello, all experts

Will the yolov3 model have the low accuracy if training small images? I rained images having a size of 50~100, but it has a low accuracy after proper epochs. Is there any expert who can help me to resolve this issue?

Thank you
